configure.ac

```diff
@@ -31,12 +31,12 @@ PKG_CHECK_MODULES(CAIRO, cairo >= $CAIRO_REQUIRED_VERSION)
 # Optional use of Pango's text rendering on top of Cairo
 AC_ARG_WITH(pango, [  --with-pango            use Pango to measure and draw text],[text_v=pango],[text_v=default])
 
-GDIPLUS_CFLAGS="$GDIPLUS_CFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -std=gnu99"
+GDIPLUS_CFLAGS="$GDIPLUS_CFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-typedef-redefinition -std=gnu99"
```

Comment:
@akoeplinger I've resolved to fix this by moving to unconditionally prevent this warning

Reply:
ok that works too but as I mentioned in the other thread we should be fine to bump to use C11 :)